Unit2_Applications/friendship-quiz.js

```diff
@@ -0,0 +1,33 @@
+
+var questionsList = ["What are your/your friend's favorite foods?", "What are your/your friend's favorite animals?", "What are your/your friend's favorite hangout spots?", "What do you/your friend's do when you're together?","Do you/your friend prefer chocolate or vanilla?","What is the best sport?"];
+function init() {
+  var randomIndex = Math.floor(Math.random() * questionsList.length);
+  document.getElementById("question").innerHTML = questionsList[randomIndex];
+}
+
+window.onload = init;
+var answerList = [];
+function submitAnswer() {
+  var answer = document.getElementById("answerInput").value;
+  var answer2 = answer.toUpperCase();
+  document.getElementById("answerInput").value = "";
+  answerList.push(answer2);
+}
+
+function displayMessage (message) {
+  document.getElementById("result").innerHTML = message
+}
+
+function checkGuess() {
+  var guess = document.getElementById("guessInput").value;
+  var guess2 = guess.toUpperCase();
```

Comment:
These two lines are quite similar to the first two lines of `submitAnswer`. How could you avoid the repeated code?